liveblog.php

```diff
@@ -236,11 +238,19 @@ private static function register_embed_handlers() {
 	 */
 	public static function init() {
 		/**
-		 * Add liveblog support to the 'post' post type. This is done here so
-		 * we can possibly introduce this to other post types later.
+		 * Add liveblog support to the 'post' and 'page' post type.
+		 */		
+		$post_types = array( 'post', 'page' );
+		
+		/**
+		 * This allows the users to filter their supported post types.
 		 */
-		add_post_type_support( 'post', self::key );
+		self::$supported_post_types = apply_filters( 'modify_supported_post_types', $post_types );
```

Comment:
Filters have to be prefixed. [Some useful info here](https://vip.wordpress.com/documentation/code-review-what-we-look-for/#prefixing-functions-constants-classes-and-slugs).